backend/src/routes/encounter.route.ts

```diff
@@ -2,12 +2,13 @@
  * Route configures endpoints, and attaches controller as an action to each route's REST methods
  */
  import { Router } from 'express';
- import {createEncounter, updateEncounter} from '../controllers/encounter.controller';
+ import {createEncounter, deleteEncounters, updateEncounter} from '../controllers/encounter.controller';
  
  const routes = Router();
  
  routes.post('/', createEncounter);
- routes.put('/:id', updateEncounter)
+ routes.put('/:id', updateEncounter);
+ routes.delete(':/id', deleteEncounters)
```

Comment:
is this meant to be `:/id` for the `routes.delete`?

Reply:
Yeah I think it should be `/:id` right?

Reply:
Ah whoops yeah